test/Tests.Infrastructure/ClusterTestBase.cs

```diff
@@ -65,6 +67,13 @@ protected DocumentStore GetDocumentStoreForRollingIndexes(Options options = null
             return base.GetDocumentStore(options, caller);
         }
 
+        public async Task<PutIndexResult> CreateIndex(IDocumentStore store, AbstractIndexCreationTask index)
+        {
+            var results = await store.Maintenance.ForDatabase(store.Database).SendAsync(new PutIndexesOperation(index.CreateIndexDefinition()));
```

Comment:
1. It should stay where it was if possible
2. It should execute `await _parent.Cluster.WaitForRaftIndexToBeAppliedInClusterAsync` internally

Reply:
fine, `Cluster.CreateIndexInClusterAsync(...)`

Reply:
It shouldn't be a static method instead?